packages/api/src/Api.ts

```diff
@@ -33,18 +33,43 @@ const Types = require('@cennznet/types');
 
 export class Api extends ApiPromise {
     static async create(options: ApiOptions | ProviderInterface = {}): Promise<Api> {
-        return new Api(options).isReady;
+        return new Api(options).isReady as Promise<Api>;
```

Comment:
the return type already have `Promise<Api>`, my ide can recognize it correctly. type casting doesn't seem necessary

Reply:
`isReady` returns a `Promise<ApiPromise>` (and `ApiPromise` does not have a `cennzxSpot` or `genericAsset` property), so I believe the cast is necessary.

Reply:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/39037239/59311833-029e7900-8cff-11e9-9fe6-059ccc1b0458.png">
create() has a return type defined.

Reply:
Yes, it does. But that still doesn't negate the need for this cast.

See:

![Screenshot from 2019-06-12 10-52-45](https://user-images.githubusercontent.com/28038653/59312221-71300680-8d00-11e9-8f52-987e5d975d1b.png)

Reply:
i don't use vs-code, but sure we can add this if it helps.

Reply:
Just to be clear, this isn't VSCode specific.

The `isReady` method returns a `Promise<ApiPromise>`. The `create()` method specifies a return type of `Promise<Api>`. `Api` contains properties that `ApiPromise` does not, which is why the case is **required**.